java/open-existing-map/src/main/java/com/esri/arcgisruntime/sample/openexistingmap/MainActivity.java

```diff
@@ -74,31 +74,31 @@ protected void onCreate(Bundle savedInstanceState) {
   }
 
   private void addDrawerItems() {
-    String[] webmapTitles = { getResources().getString(R.string.webmap_houses_with_mortgages_title),
-        getResources().getString(R.string.webmap_usa_tapestry_segmentation_title),
-        getResources().getString(R.string.webmap_geology_us_title) };
+    String[] webmapTitles = { getResources().getString(R.string.webmap_geology_us),
+        getResources().getString(R.string.webmap_terrestrial_ecosystems),
+        getResources().getString(R.string.webmap_hurricanes_cyclones_typhoons) };
     ArrayAdapter<String> mAdapter = new ArrayAdapter<>(this, android.R.layout.simple_list_item_1, webmapTitles);
     mDrawerList.setAdapter(mAdapter);
 
     mDrawerList.setOnItemClickListener((adapterView, view, position, id) -> {
       if (position == 0) {
-        mPortalItem = new PortalItem(mPortal, getResources().getString(R.string.webmap_houses_with_mortgages_id));
+        mPortalItem = new PortalItem(mPortal, getResources().getString(R.string.webmap_geology_us_id));
```

Comment:
let's just hardcore the ids for these three and not both keeping them in strings.

This was an old pattern we used to use because of agressive linting from Android. No one's going to translate an id (what the R.string resources are for), so let's just hardcode them in place

Reply:
Sounds good!